### PR TITLE
Fix extraction bug on move when redis is active.

### PR DESCRIPTION
**Pull request by jone**
When redis is active for deferred extraction in a multi instance setup, the move job may contain invalid paths when the extraction job is processed in a different process than the one the job was created.

The problem is that the event infos are stored on the object until the extraction happened. Since the event infos contain acquisition wrapped objects and the deferreing results in that the extraction happens in a different transaction, it caused persistent pointers to be stored in the database.
The extraction did rely on having acquisition wrapped objects, which was sometimes not the case, depending on the database cache state.

The solution is to not store anything but prepare strings early.
The event infos are now stored in an `additional_data` dict on the job, so that it can be reused on extration time.
Since the deferred extraction has no reference to the job object, the infos are passed to the worker as serialized json.

Since there was no move test at all I added one testing the extracted result. It did work before my changes, my changes did not change the result.
I have reproduced the issue in a multi instance redis setup reliably and could profe that this change fixes the problem by inspecting the paths in the extracted JSON file; they are now the correct relative paths.
I have verified that the move extraction with redis still works with this changes.

This closes #49 because it fixes this issue; relates to #45, #47.

### CODE REVIEW

File: ftw/publisher/sender/browser/views.py [hunk 1]
```diff
@@ -127,24 +128,33 @@ def __call__(self, no_response=False, msg=None, *args, **kwargs):
...
+            self.context.Title(),
+            '/'.join(self.context.getPhysicalPath()),
+        ))
         # status message
         if msg is None:
             msg = _(u'Object move/rename action has been added to the queue.')
-        IStatusMessage(self.request).addStatusMessage(
-            msg,
-            type='info'
+            IStatusMessage(self.request).addStatusMessage(
```
Comment (mbaechtold) [inline ftw/publisher/sender/browser/views.py]: What's the reason for this change? What happens if `msg` is passed as an argument to this method?

Comment (jone) [inline ftw/publisher/sender/browser/views.py]: Opps, that's an accident, thanks for pointing out!
(caused by emacs' auto-format)
